doc/en/components/grids/_shared/validation.md

```diff
@@ -0,0 +1,672 @@
+---
+title: Editing and Validation in {Platform} {ComponentTitle} - Infragistics
+_description: Validate the input of the users in grid and notify them if it's valid or not while using {Platform} {ComponentTitle}. See demos & examples!
+_keywords: {Platform} validation, ignite ui for {Platform}, infragistics
+---
+
+# {Platform} {ComponentTitle} Editing and Validation
+The {ComponentTitle}'s editing exposes a built-in validation mechanism of user input when editing cells/rows. It extends the [{Platform} Form validation](https://{Platform}.io/guide/form-validation) functionality to allow easier integration with a well known functionality. When the state of the editor changes, visual indicators are applied to the edited cell.
+
+## Configuration
+
+### Configure via template-driven configuration
+
+We extend some of the {Platform} Forms validator directives to directly work with the `IgxColumn`. The same validators are available as attributes to be set declaratively in `igx-column`. The following validators are supported out-of-the-box:
+- required
+- min
+- max
+- email
+- minlength
+- maxlength
+- pattern
+
+To validate that a column input would be set and the value is going to be formatted as an email, you can use the related directives:
+```html
+<igx-column [field]="email" [header]="User E-mail" required email></igx-column>
+```
+
+The following sample demonstrates how to use the prebuilt `required`, `email` and `min` validator directives in a {ComponentTitle}.
+<!-- ComponentStart: Grid -->
+<code-view style="height:600px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/grid-validator-service" alt="{Platform} {ComponentTitle} Validation Basic Example">
+</code-view>
+<!--ComponentEnd:Grid-->
+
+<!-- ComponentStart:TreeGrid -->
+<code-view style="height:600px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/tree-grid-validator-service" alt="{Platform} {ComponentTitle} Validation Basic Example">
+</code-view>
+
+<!-- ComponentEnd:TreeGrid -->
+
+<!-- ComponentStart:HierarchicalGrid -->
+<code-view style="height:680px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-validator-service" alt="{Platform} {ComponentTitle} Validation Basic Example">
+</code-view>
+
+<!-- ComponentEnd:HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.